File: jm-redis-cacheable/src/main/java/com/example/jmrediscacheable/serve/service/UserService.java
package com.example.jmrediscacheable.serve.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.example.jmrediscacheable.serve.domain.User;

public interface UserService extends IService<User> {

    User getUser(Long id);

    Boolean updateUser(User user);
}

File: kg-redis-cacheable/src/main/kotlin/com/example/kgrediscacheable/serve/service/UserService.kt
package com.example.jgrediscacheable.serve.service

import com.baomidou.mybatisplus.extension.service.IService
import com.example.kgrediscacheable.serve.domain.User

/**
 * @author : songtc
 * @since : 2023/12/08 14:16
 */
interface UserService : IService<User> {
    fun getUser(id: Long): User

    fun updateUser(user: User): Boolean
}